src/app/(proper_react)/redesign/(authenticated)/user/dashboard/View.tsx

```diff
@@ -116,17 +118,18 @@ export const View = (props: Props) => {
   const tabSpecificExposures = getTabSpecificExposures(selectedTab);
   const filteredExposures = filterExposures(tabSpecificExposures, filters);
   const exposureCardElems = filteredExposures.map((exposure: Exposure) => {
-    const exposureCardKey = `${isScanResult(exposure) ? "scan" : "breach"}-${
-      exposure.id
-    }`;
+    const exposureCardKey = isScanResult(exposure)
+      ? "scan-" + exposure.onerep_scan_result_id
+      : "breach-" + exposure.id;
     return (
       <li key={exposureCardKey} className={styles.exposureListItem}>
+        {exposure.id}
         <ExposureCard
           exposureData={exposure}
           isExpanded={exposureCardKey === activeExposureCardKey}
-          setExpanded={() => {
+          onToggleExpanded={() => {
             if (exposureCardKey === activeExposureCardKey) {
-              setActiveExposureCardKey("");
+              setActiveExposureCardKey(null);
```

Comment:
What's the reason behind using `null` instead of `""`?

I know this PR isn't to address the "collapse/expand multiple cards" bug, but I'm thinking this might be the root cause. I think I was checking for `"" == ""`, which would always return `true` for anything that has been expanded and collapsed before. I think `null==null` check would also yield the same result and not fix the issue?

Reply:
Oh yeah, that's technically unrelated to this PR, but it was the first thing I tried and I felt it was probably a good change anyway. When using an empty string, the type system can't distinguish between "the currently expanded card" and "no card is expanded at the moment", which means we won't be warned when we're forgetting a check for the latter.

That said, I think this PR does actually address the "collapse/expand multiple cards" bug, because multiple cards would be considered to have the same ID (i.e. their scan ID, rather than that card's scan result's ID). I'm not sure why the check would be `"" === ""`, since `exposureCardKey` should never be an empty string?